client/state/analytics/middleware.js

```diff
@@ -15,12 +21,18 @@ const eventServices = {
 
 const pageViewServices = {
 	ga: ( { url, title } ) => analytics.ga.recordPageView( url, title ),
-	default: ( { url, title } ) => analytics.pageView.record( url, title )
+	'default': ( { url, title } ) => analytics.pageView.record( url, title ),
+};
+
+const loadTrackingTool = ( trackingTool, state ) => {
+	if ( trackingTool === 'Lucky Orange' && isTracking( state ) ) {
```

Comment:
Maybe we should also move [the other criteria](https://github.com/Automattic/wp-calypso/pull/12332/files#diff-d088a91c46152a3ed739a47ed406c5bbR360) here so they're all in one place.